### PR TITLE
feat(md-divider): initial divider implementation

### PR DESCRIPTION
**Pull request by Dona278**
Like in angular-material v1

### CODE REVIEW

**Comment by Dona278** [comment]
I signed it!

File: src/components/divider/divider.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    Component,
+    Directive, 
+    ViewEncapsulation    
+} from '@angular/core';
+
+/** The inset directive, used to change divider behavior from "full-bleed" to "inset". */
+@Directive({    
+    selector: 'md-inset'    
+})
+export class MdInset {}
```
Comment (DevVersion) [inline src/components/divider/divider.ts]: Why having an extra directive for that?

File: src/components/divider/divider.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+/**
+ * Component that represents a horizontal line, along with styling it according to the Material Design.
+ */
+@Component({    
+    moduleId: module.id,
+    selector: 'md-divider',    
+    template: '',
+    styleUrls: ['divider.css'],
+    encapsulation: ViewEncapsulation.None
```
Comment (DevVersion) [inline src/components/divider/divider.ts]: Indention should be with two spaces.

File: src/components/divider/divider.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+})
+export class MdDivider {}
+
+//Export directives
```
Comment (DevVersion) [inline src/components/divider/divider.ts]: Unnecessary comment IMO + there should be a space between the text and `//`, to pass the TSLint checks

File: src/components/divider/divider.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+@import 'variables';
+@import 'default-theme';
+
+md-divider {
```
Comment (DevVersion) [inline src/components/divider/divider.scss]: Not sure about that PR in general, but the divider is already implemented in the list component (as it belongs to that)
- See https://github.com/angular/material2/blob/master/src/components/list/list.scss#L156

**Comment by jelbourn** [comment]
@Dona278 We love your enthusiasm, but divider is not a component we plan on adding to the library. Be sure to check out our [contributing guidelines](https://github.com/angular/material2/blob/master/CONTRIBUTING.md#-want-a-feature) in the future so we can give you feedback before jumping into writing code.

**Comment by Dona278** [comment]
Thank you @jelbourn and @DevVersion for reply. I wrote this "micro" component because i use it often with the first version of angular material (and it's a directive and not a simple css class).

In answer to @DevVersion in the material design spec you can see an image like email composer which are input text divided by divider.

In the future i check better your contributing guidelines, promised :)

Thanks again!

**Pull request closed without merging**